kubernetes/schema_rbac.go

```diff
@@ -43,7 +43,7 @@ func rbacRoleRefSchema() map[string]*schema.Schema {
 		"api_group": {
 			Type:         schema.TypeString,
 			Description:  "The API group of the user. The only value possible at the moment is `rbac.authorization.k8s.io`.",
-			Required:     true,
+			Optional:     true,
```

Comment:
Making this required was apparently intentional: https://github.com/terraform-providers/terraform-provider-kubernetes/pull/331#discussion_r259407283

https://github.com/terraform-providers/terraform-provider-kubernetes/pull/331/files#diff-04d70cf27105c3daf3dc6d90bc6ef8e3R11-R14

If that's still intentional, it's rather the documentation that needs updating.